documentation/guidance/gsuite/account-management.md

```diff
@@ -13,6 +13,8 @@ All the G Suite user, group and organizational unit definitions are being contro
 
 This sheet provides a declarative way to specify G Suite account state. You just need to put into it the desired state of all the accounts and groups, and, once you check the `apply` box, the GitHub Action in the `infrastructure-gsuite` repository will perform the required updates through the G Suite API.
 
+> :book: Please note that, due to the lack of mail support for the spreadsheet owner account, the only ways of requesting access are through a current member or by logging in with the [owner credentials](https://start.1password.com/open/i?a=J3YBD4VIJZGH3ORXYSVY6UUFXY&h=helpfulengineering.1password.com&i=nmlfxmdqp5ejzbb42x5fanz3p4&v=brpvigfbzutlo5cnfvolyeikji) and inviting yourself.
```

Comment:
This [owner credentials](https://start.1password.com/open/i?a=J3YBD4VIJZGH3ORXYSVY6UUFXY&h=helpfulengineering.1password.com&i=nmlfxmdqp5ejzbb42x5fanz3p4&v=brpvigfbzutlo5cnfvolyeikji) link takes me to the Terraform admin credentials, is that correct?

I would have assumed the payer account is the/an owner? If that is the case, with 2FA turned on people still wouldn't be able to log into the account to grant access as the text or phone call would go to the individual who set up the account.

Reply:
> This [owner credentials](https://start.1password.com/open/i?a=J3YBD4VIJZGH3ORXYSVY6UUFXY&h=helpfulengineering.1password.com&i=nmlfxmdqp5ejzbb42x5fanz3p4&v=brpvigfbzutlo5cnfvolyeikji) link takes me to the Terraform admin credentials, is that correct?

Yes, indeed: that user not only has the required privileges to programmatically modify the infrastructure, but also owns the spreadsheet and the service account.

> I would have assumed the payer account is the/an owner? If that is the case, with 2FA turned on people still wouldn't be able to log into the account to grant access as the text or phone call would go to the individual who set up the account.

Unfortunately, we can't specify multiple spreadsheet owners, so I chose to group all the G Suite Terraform-related stuff under a single purpose-specific account. I think that using the legacy "superuser" payer account would negatively impact both privilege separation and clarity.

The owner account doesn't have phone-based 2FA, but TOTP. You can retrieve  fresh six-digit codes from 1Password:

![Screenshot_20200809-145023749 (1).jpg](https://user-images.githubusercontent.com/11387611/89732584-00919080-da50-11ea-8bb8-5922c535a7a8.jpg)